components/Dialogs/src/MDCDialogPresentationController.m

```diff
@@ -146,6 +137,20 @@ - (void)presentationTransitionWillBegin {
   // presentation and dismissal of the dialog.
   // https://material.io/guidelines/motion/choreography.html#choreography-creation
 
+  // Ensure corner radius matches between the tracking view and the view being presented
```

Comment:
_Question:_ Can we remove the `_trackingView` all together?

Reply:
I think that's the view that renders the shadow, so it's needed for the presentation.